apps/MUXSDKStatsExampleSPM/MUXSDKStatsExampleSPM.xcodeproj/project.pbxproj

```diff
@@ -529,7 +529,7 @@
 				CURRENT_PROJECT_VERSION = 1;
 				DEVELOPMENT_TEAM = XX95P4Y787;
 				GENERATE_INFOPLIST_FILE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 17.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 14.0;
```

Comment:
question(non-blocking): Assuming this is what it should have been all along and we support back to at least iOS v14?

Reply:
Good question, we support back to iOS 11, but the older versions require the CI to download older iOS runtimes so they are not tested